app/src/main/scala/org/alephium/explorer/util/TimeUtil.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.util
+
+import java.time._
+
+object TimeUtil {
```

Comment:
great, I have few functions to move here too :rocket:

Reply:
I will add some black magic here later to improve the performance of `TimeStamp.now` when it becomes a bottleneck :)